### ISSUE TITLE
my firefox 126.0 (64-bit) version does not executes gesturefly

### ISSUE THREAD

**Issue by aurelio20000** [Robbendebiene/Gesturefy#716]
### Describe your matter in detail
<!-- i am trying to use gesturefly it doesent works on 126.0 (64-bit) firefox. 

### Your System
- Operating system: windows 10
- Firefox version: 126.0 (64-bit)
- Gesturefy version:  Gesturefy, Version: 3.2.12

**Comment by aurelio20000** [comment]
Gesture fly  Gesturefy, Version: 3.2.12 does not works on firefox 126.0 64version. Neither gesturefly or other similar add ons

**Comment by Robbendebiene** [comment]
Have you followed https://github.com/Robbendebiene/Gesturefy/wiki/Troubleshooting ?

What does "not work" mean to you? With this little information I'm unable to help you.

> or other similar add ons

This already sounds like it is not a problem caused by Gesturefy.

**Comment by Jay75043** [comment]
I don't know if it's related but a new problem just started for me where the context menu pops up.  Gesturefy has been working great for me for a long while.  After trying all sorts of troubleshooting ideas I ultimately tried a different mouse and that "fixed" the problem.  The mouse that exhibits the issue is a Logitech m570 trackball.  I tested two other mice (also Logitech) and both work fine.  I uninstalled the Logitech SetPoint software thinking perhaps that was the problem and uninstalled/reinstalled all of the mice using the device manager but nothing fixed the issue with the m570.  

I did just do a Windows update so I suspect the issue is related to that.
